src/main/data/studios.json

```diff
@@ -11,6 +11,10 @@
     "code": "9LE",
     "description": "9 LIGHT ENTERTAINMENT"
   },
+  {
+    "code": "AD",
+    "description": "ART DOYDU"
+  },
```

Comment:
You can now add in the URL and contact info. Use this as an example:

{
    "code": "1VP",
    "contact": {
      "address": "Feiteng Art Space Zoom A, No.173,Caochangdi, Chaoyang district, Beijing, China",
      "email": "mali@123vp.com",
      "name": "Mr.Ma"
    },
    "description": "123 Vision Pictures",
    "url": "https://www.123vp.com"
  },